packages/did-core-test-server/src/services/scenarios/equivalence-properties/equivalence-properties-tests.test.js

```diff
@@ -0,0 +1,48 @@
+const request = require('supertest');
+
+const { app } = require('../../../app');
+
+const fixture = require('../../../__fixtures__').scenarios['equivalence-properties'];
+let api;
+
+beforeAll(async () => {
+  await app.ready();
+  api = request(app.server);
+});
+
+afterAll(async () => {
+  await app.close();
+});
+
+it('should should test that both DID URIs are from the resolving DID method', async () => {
+  const response = await api
+    .post('/test/scenarios')
+    .set('Accept', 'application/json')
+    .send(fixture.equivalence.request);
+  // uncomment to align with fixtures
+  // console.log(JSON.stringify(response.body, null, 2));
+  expect(response.status).toBe(200);
+  expect(response.body).toEqual(fixture.equivalence.response);
+});
+
+// it('should should match test fixture expectations', async () => {
```

Comment:
You may not need these other tests, if you embed all the data needed for the assertions in a single fixture.